packages/tree-extension/src/index.ts

```diff
@@ -3,19 +3,19 @@
 
 import {
   JupyterFrontEnd,
-  JupyterFrontEndPlugin
+  JupyterFrontEndPlugin,
```

Comment:
Were these prettier changes added automatically on commit?

Reply:
Yes, I couldn't commit because of prettier check, so I ran `jlpm prettier` and add the changes.
But the command did have change a lot of files, with the same kind of modification.

Reply:
Thanks @jtpio, I had some weird file in `node-modules/.bin`.
`jlpm clean` did the trick.